### DISCUSSION TITLE
https://soundgrail.com/blog/how-social-media-and-ugc-could-decide-the-future-of-the-music-industry

### DISCUSSION THREAD

**Discussion by giscus** [Distalk/giscus#30] [Giscus]
# https://soundgrail.com/blog/how-social-media-and-ugc-could-decide-the-future-of-the-music-industry

Thunder only happens when it's raining. Social media and user-generated content (UGC) could determine the future of the music industry.

https://soundgrail.com/blog/how-social-media-and-ugc-could-decide-the-future-of-the-music-industry

**Comment by sofis-har** [comment]
Social networks are very popular now. But sometimes the lack of subscribers and likes is very upsetting. It’s good that it is possible to purchase https://promosoundgroup.net/collection/social-promotion/new-instagram-promo/ real subscribers and likes.

**Comment by Mateorosi** [comment]
Useful article!

**Comment by Mateorosi** [comment]
Totally feel this. Social media basically made half the artists we listen to now. Like, one viral TikTok and boom—record deal. Even fans calling out labels on YouTube complaints https://youtube.pissedconsumer.com/customer-service.html  is shaking things up. It’s wild how UGC and comments can push a song or destroy a rep overnight. Kinda scary, but also super empowering for indie artists.
